.vscode/settings.json

```diff
@@ -0,0 +1,36 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  "prettier.tabWidth": 4,
+  "prettier.bracketSpacing": true,
+  "prettier.printWidth": 120,
```

Comment:
👍  to this, I also prefer 120 with prettier. 80 tends to mangle stuff a little too much for my taste.